.postcssrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "parser": "sugarss",
+  "map": false,
+  "plugins": {
+    "postcss-plugin": {}
+  }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/6414141/90544656-720ac680-e155-11ea-8fb7-c766416bc4b3.png)

Also in _datePickerFrame.css, _defaults.css, _structure.css, and _virtualDOM.css.